contrib/localnet/orchestrator/smoketest/test_stress.go

```diff
@@ -60,7 +60,8 @@ func init() {
 	StressCmd.Flags().StringVar(&stressTestArgs.zevmURL, "zevmURL", "http://zetacore0:8545", "--zevmURL http://zetacore0:8545")
 	StressCmd.Flags().StringVar(&stressTestArgs.deployerAddress, "addr", "0xE5C5367B8224807Ac2207d350E60e1b6F27a7ecC", "--addr <eth address>")
 	StressCmd.Flags().StringVar(&stressTestArgs.deployerPrivateKey, "privKey", "d87baf7bf6dc560a252596678c12e41f7d1682837f05b29d411bc3f78ae2c263", "--privKey <eth private key>")
-	StressCmd.Flags().BoolVar(&stressTestArgs.local, "local", true, "--local")
+	StressCmd.Flags().StringVar(&stressTestArgs.network, "network", "PRIVNET", "--network PRIVNET")
```

Comment:
I don't know if we use privnet naming anywhere else but I think localnet is a more common language to specify the local network

Reply:
I agree it isn't the correct language for this case but it is the name of the build flag used for the smoketest and config files.